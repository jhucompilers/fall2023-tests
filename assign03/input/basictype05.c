int main(void) {
  // If a declaration uses a basic type, but omits "char", "int", and "void",
  // "int" is assumed. So the following are equivalent to "signed int"
  // and "unsigend int".
  signed s;
  unsigned u;

  s = -2000000000;
  u = 3000000000;

  return 0;
}
