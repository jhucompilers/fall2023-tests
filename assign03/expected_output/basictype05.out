0|main|function|function () returning int
2|s|variable|int
2|u|variable|unsigned int
